makefile

```
@@ -23,137 +26,52 @@ ifeq ($(UNAME), Darwin) # MacOS
 	base64=base64
 endif
 
-.PHONY: help
-help:
-ifeq ($(UNAME), Linux)
-	@echo "git clone https://github.com/cosmos/cosmos-sdk -> git checkout $(COSMOS_SDK_VERSION) -> make build -> mv build/simd $HOME/go/bin/"
-endif
-ifeq ($(UNAME), Darwin) # macOS
-	@echo "git clone https://github.com/cosmos/cosmos-sdk -> git checkout $(COSMOS_SDK_VERSION) -> make build -> echo $GOPATH/bin sudo mv build/simd /usr/local/go/bin/ -> ln -s /usr/local/go/bin/simd /usr/local/bin/simd"
-endif
+.PHONY: install-local-ledger
+install-local-ledger:
+	rm -rf local-ledger
+	git clone --depth 1 --branch $(LEDGER_BRANCH) https://github.com/regen-network/regen-ledger.git local-ledger/temp
+	cd local-ledger/temp && yarn install && make build
```

I don't think we have a `yarn install` in the ledger repo
```suggestion
	cd local-ledger/temp && make build
```